packages/admin-portal/src/components/pages/devs-management/devs-management.tsx

```diff
@@ -181,49 +180,38 @@ export const DevsManagement: React.FC = () => {
     },
   ]
 
-  if (!isLoading && data?.length === 0) {
-    return (
-      <React.Fragment>
-        <Helper variant="info">
-          Unfortunately, there are no results that match your search criteria, please try again
-        </Helper>
-        <Link className="text-center" to={Routes.DEV_MANAGEMENT}>
-          <Button variant="primary" type="button">
-            New Search
-          </Button>
-        </Link>
-      </React.Fragment>
-    )
-  }
-
   return (
```

Comment:
I think
```
if (isLoading) {
  return <Loader />
}
```

Reply:
if we render the `Loader` here, the page will rerender the filter form when fetching data, I think it not necessary. I think I move the Loader to `renderResult` function for more readable.